doc/final_report.md

```diff
@@ -0,0 +1,90 @@
+
+```{r groups_sentence_distributions, echo=FALSE, fig.cap="Figure 1. The distribution of the aggregate sentence time of indigenous and non-indigenous inmates.”, out.width = '100%'}
+
+knitr::include_graphics("../results/“DISTRIBUTION.png")
+```
+
+Looking at the boxplot, we can see that there is a large overlap in the confidence intervals between the two groups, and potentially indicates that there is not a significant. 
+
+```{r groups_sentence_distributions, echo=FALSE, fig.cap="Figure 2. The distribution of the aggregate sentence time of indigenous and non-indigenous inmates.”, out.width = '100%'}
+
+knitr::include_graphics("../results/“DISTRIBUTION.png")
+```
+
+As we selected the median, we subsequently carried out a permutation test under the null hypothesis computationally with `r N_REPEATS`.  This model is applicable as we have a large sample size.
+
+The null hypothesis stated that there is no difference in the median sentence lengths between the indigenous and non-indigenous inmates. The alternate hypothesis stated that there is a difference in the median sentence lengths between the indigenous and non-indigenous populations. The test statistic was the difference in the sample medians of the two groups. The significance level was set to 0.05. 
+
+The distribution under the null hypothesis has the expected normal shape due to the sample size and number of repetitions.
+
+```{r groups_sentence_distributions, echo=FALSE, fig.cap="Figure 3. The null distribution generated using permutation of the difference in medians of the two groups.”, out.width = '100%'}
+
+knitr::include_graphics("../results/“NULL_DISTRIBUTION.png")
+```
+
+The difference in the sample test medians was `r DIFF_IN_MEANS`, which corresponded to a p-value of  `r P_VALUE`. As the p-value was greater than the significance level of 0.05, we do not have enough evidence to reject the null hypothesis. Hence there is no statistically significant difference in the median sentence lengths between indigenous and non-indigenous groups. 
+
+```{r observations_group, echo=FALSE}
+knitr::kable(model_quality$table, caption = "Table 2. The summary of the hypothesis test”)
+```
+
+Though the study revealed that there is no statistical significance, the practical takeaway needs to be assessed in terms of the effect size and sample size. As our samples had such a small difference in the medians, it is likely that the effect size of the test is very small. This increases the risk that we may have committing Type-II error, where we may have missed rejecting the null hypothesis. However, we did have a very large sample size for both groups, which acts as a balance to this small effect size as this reduces the risk of a Type-II error. 
+
+A follow-up study to this could combine the inmates data from multiple years to increase the sample size, and potentially conduct the hypothesis test with a larger alpha value.
```

Comment:
A follow-up study to this could combine the inmates data from multiple years to increase the sample size, and potentially conduct the hypothesis test with a larger alpha value. Additionally, a study that controls for other variables, most notably the nature of the offence, could give us better and more nuanced understanding of the the possible correlations and dynamics.